Generalizing content on learn/tanssi/included-templates (#192)
moondance-labs/tanssi-docs: learn/tanssi/included-templates.md

### Baseline Appchain Template {: #baseline-appchain-template }

As presented in the [Overview](/learn/tanssi/overview/){target=\_blank} article, appchains deployed through Tanssi are fully sovereign and customizable blockchains.

> As part of the Tanssi ecosystem, appchains must include the essential components to implement the consensus mechanism and be able to interact and synchronize with the security provider. The Baseline Tanssi appchain Template includes all the necessary functionality for the block producers logic, p2p, database, and synchronization layers between the appchain and the security provider, allowing developers to focus solely on customizing their product.

Reviewer: ```suggestion
As part of the Tanssi ecosystem, appchains must include the essential components to implement the consensus mechanism and be able to interact and synchronize with the security provider of their choice (for example, Symbiotic on Ethereum). The baseline Tanssi appchain Template includes all the necessary functionality for the block producers logic, p2p, database, and synchronization layers between the appchain and the security provider, allowing developers to focus solely on customizing their product.
```

This template also includes Tanssi's [Author Noting](https://github.com/moondance-labs/tanssi/blob/master/pallets/author-noting/src/lib.rs){target=\_blank} module, which implements the logic for retrieving and validating the set of sequencers assigned to provide block production services to the appchain. It also includes logic that allows a sequencer to sign the block when the consensus mechanism determines that it is the sequencer's turn to produce the block (and thus be rewarded accordingly).

The source code for this template is public and accessible on the [Tanssi GitHub repository](https://github.com/moondance-labs/tanssi/blob/master/container-chains/runtime-templates/simple/src/lib.rs){target=\_blank}.

### Baseline EVM (Ethereum Virtual Machine) Template {: #baseline-evm-template }

> Extending the [Baseline Tanssi appchain template](#baseline-appchain-template), this template provides not only Tanssi protocol support but also an EVM and full Ethereum compatibility.

Reviewer: ```suggestion
Extending the [baseline Tanssi appchain template](#baseline-appchain-template), this template provides not only Tanssi protocol support but also an EVM and full Ethereum compatibility.
```

> Leveraging the [Frontier project](https://github.com/paritytech/frontier){target=\_blank}, this template includes an Ethereum compatibility layer for appchains to allow running unmodified Ethereum dApps.

Reviewer: ```suggestion
Leveraging a set [EVM-specific modules](https://github.com/paritytech/frontier){target=\_blank}, this template includes an Ethereum compatibility layer for appchains to allow running unmodified Ethereum dApps.
```
